libs/execution_context/ebpf_program.c

```diff
@@ -397,6 +397,30 @@ ebpf_program_type(_In_ const ebpf_program_t* program)
     return &ebpf_program_get_parameters(program)->program_type;
 }
 
+ebpf_result_t
+ebpf_program_associate_map(ebpf_program_t* program, ebpf_map_t* map)
```

Comment:
suggest: there is an ebp_program_associate_maps (plural) already. So can the name be changed to reduce confusion and error in future? How about using _single_map  or _additional_map ?

Reply:
Done